frontend/src/scenes/session-recordings/SessionPlayerDrawer.tsx

```diff
@@ -29,7 +28,7 @@ export function SessionPlayerDrawer({ isPersonPage = false, onClose }: SessionPl
                     justify="space-between"
                 >
                     <Button type="link" onClick={onClose}>
-                        <ArrowLeftOutlined /> Back to {isPersonPage ? 'persons' : 'recordings'}
+                        <ArrowLeftOutlined /> Back
```

Comment:
Nice I like the generalization here. Thought this was too specific in the first place.